internal/ingress/controller/nginx.go

```diff
@@ -602,6 +615,31 @@ func (n *NGINXController) OnUpdate(ingressCfg ingress.Configuration) error {
 
 	cfg.SSLDHParam = sslDHParam
 
+	for _, server := range ingressCfg.Servers {
```

Comment:
Let's do this feature in stages. Please remove this section. For this PR SSL should be handled by NGINX, not LUA.